### PR TITLE
Help panel

### LINKED ISSUE #119: Create help panel
When a user clicks on the help button, the help panel should be visible

### PR DESCRIPTION
**Pull request by lesliexin**
Closes #119.

image
image

### CODE REVIEW

File: ui/src/components/HelpAccordian.tsx [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+const StyledAccordian = styled(Accordion)`
+  && {
+    box-shadow: none;
+  }
+
+  .MuiAccordionSummary-root {
+    padding: 0px;
+  }
+
```
Comment (hanlinc27) [inline ui/src/components/HelpAccordian.tsx]: Not sure which class to change here but can you add an `overflow: hidden` to get rid of the horizontal scroll?

image

File: ui/src/styles/typography.ts [hunk 2]
```diff
@@ -14,6 +14,16 @@ export const fontSize = Object.freeze({
   interactive: "16px",
 });
 
+export const Body1 = styled.span`
+  font-size: ${fontSize.subtitle};
```
Comment (hanlinc27) [inline ui/src/styles/typography.ts]:
```suggestion
```

File: ui/src/components/HelpAccordian.tsx [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+  .MuiAccordionSummary-content {
+    margin: 10px 0px 10px 20px;
+  }
+
+  .MuiAccordionSummary-content.Mui-expanded {
+    margin: 10px 0px 10px 20px;
+    color: ${colors.primaryDark1};
```
Comment (hanlinc27) [inline ui/src/components/HelpAccordian.tsx]: It's a small thing but our primaryDark1 is the wrong dark teal, it should be `#2D6394` according to Figma but right now it's `#2D8694` which is leading to a discrepancy
image
vs.
image

File: ui/src/components/HelpDrawer.tsx [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+export function HelpDrawer({
+  state,
+  setIsHelpDrawerOpen,
+  setIsTutorialOpen,
+}: HelpDrawerProps): JSX.Element {
+  const handleLaunchTour = () => {
+    setIsTutorialOpen(TutorialState.First);
+    setIsHelpDrawerOpen(HelpDrawerState.Closed);
+  };
+
```
Comment (hanlinc27) [inline ui/src/components/HelpDrawer.tsx:104]: I think the zoom controls need to move left to still be visible when the panel is open?
image

File: ui/src/components/HelpAccordian.tsx [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+    box-shadow: none;
+  }
+
+  .MuiAccordionSummary-root {
+    padding: 0px;
+  }
+
+  .MuiAccordionSummary-content {
+    margin: 10px 0px 10px 20px;
+  }
```
Comment (hanlinc27) [inline ui/src/components/HelpAccordian.tsx]:
```suggestion
  &:hover {
      color: ${colors.primaryDark1};
    }
  }
```
image

**Review by hanlinc27 — CHANGES_REQUESTED** [review]
It looks really good/clean! Just some small changes 😄👍

File: ui/src/components/HelpAccordian.tsx (hunk 5 above)
Comment (hanlinc27) [reply ui/src/components/HelpAccordian.tsx]: Didn't comment it but also for the dropdown arrow icon ^^

File: ui/src/components/HelpDrawer.tsx (hunk 4 above)
Comment (lesliexin) [reply ui/src/components/HelpDrawer.tsx:104]: Ooh yes I forgot about that!

File: ui/src/components/HelpAccordian.tsx (hunk 3 above)
Comment (lesliexin) [reply ui/src/components/HelpAccordian.tsx]: Yeah, I was wondering this as well. The Figma colour styles are #2D6394 but on the design system page it says #2D8694.

**Review by AbhiPrasad — APPROVED** [review]

**Review by hanlinc27 — APPROVED** [review]

**Pull request merged**